Spanner/tests/System/PgTransactionTest.php

```diff
@@ -92,7 +92,10 @@ public function testTransactionNoCommit()
         $ex = false;
         try {
             $db->runTransaction(function ($t) {
-                $t->execute('SELECT * FROM ' . self::$tableName);
+                $result = $t->execute('SELECT * FROM ' . self::$tableName);
+                foreach ($result as $row) {
+                    break;
```

Comment:
The `rows()` function in the `Result` class is where the callable gets called and the transactionId gets generated. Since Result is an Iterator, added these steps to fetch the transactionId. There is no other way to do this!

Reply:
Removed this change due to the current implementation alternative.